web/src/stream_data.ts

```diff
@@ -565,10 +565,6 @@ export function can_unsubscribe_others(sub: StreamSubscription): boolean {
         return false;
     }
 
-    if (current_user.is_admin) {
-        return true;
-    }
```

Comment:
Hmm, I see that's what we've implemented in the server, but can you start a #design conversation about whether that's what we want (Or bump the existing one)? I'm not entirely sure I feel confident in this detail.

Reply:
https://chat.zulip.org/#narrow/channel/101-design/topic/permissions.20for.20admin.20to.20unsubscribe.20others

Reply:
- [ ]